query-engine/connector-test-kit/src/test/scala/writes/ids/IntIdCreateSpec.scala

```diff
@@ -126,30 +126,4 @@ class IntIdCreateSpec extends FlatSpec with Matchers with ApiSpecBase {
     result.pathAsString("data.createTodo.title") should equal("the title")
     result.pathAsLong("data.createTodo.id") should equal(1)
   }
-
-  "Creating an item with an id field of type Int with autoincrement and providing an id" should "error" in {
-    val project = ProjectDsl.fromString {
-      s"""
-         |model Todo {
-         |  id    Int @id @default(autoincrement())
-         |  title String
-         |}
-       """.stripMargin
-    }
-    database.setup(project)
-
-    server.queryThatMustFail(
-      """
-        |mutation {
-        |  createTodo(data: { title: "the title", id: 2}){
-        |    id
-        |    title
-        |  }
-        |}
-      """.stripMargin,
-      project,
-      0
-    )
-  }
-
```

Comment:
Did this move to a different file or is that test case indeed gone?

Reply:
Removed test.

Reply:
I thought a bit about it and can bring the test back in a different form I believe. It's a bit harder because you need to "trigger" the right union type to parse.

Reply:
Ah only now I understand why this test case is gone. The new unchecked API allows to write to every scalar field and does not have any safeguard checks. Hence users are now able to write those id fields if they indeed want to.